python/llm/src/ipex_llm/transformers/npu_models/common.py

```diff
@@ -75,10 +75,11 @@ def split_linears(module: torch.nn.Module, n_splits_hidden_size=2, n_splits_down
     from transformers.models.qwen2.modeling_qwen2 import Qwen2MLP, Qwen2Attention
     from transformers.models.llama.modeling_llama import LlamaMLP, LlamaAttention
     attn_module_names = ["q_proj", "k_proj", "v_proj", "o_proj"]
+    baichuan_attn_module_names = ["W_pack", "o_proj"]
     mlp_module_names = ["down_proj", "up_proj", "gate_proj"]
     if (
         isinstance(module, (Qwen2Attention, LlamaAttention))
-        or module.__class__.__name__ in ['MiniCPMAttention', 'Attention']
+        or module.__class__.__name__ in ['MiniCPMAttention']
```

Comment:
Shall we also update the following check of `module.__class__.__name__ in ['MiniCPMMLP', 'MLP']`?